default-parameters-es6/demo.js

```diff
@@ -4,7 +4,7 @@
 function greetES5(message, to) {
   // Test parameter values in the function body, specifying a default
   to = to || 'DOM';
-  // OR if (typeof to === undefined) { to = 'DOM'; }
+  // OR if (typeof to === 'undefined') { to = 'DOM'; }
```

Comment:
i think `undefined` is right, because comment says, if `to` is undefined then `to = 'DOM'`. means if `to` is falsy value `undefined` then  assign `'dom'` into `to`.